### ISSUE TITLE
Strange small numbers at intersections; no matching data on OSM.org

### ISSUE THREAD

**Issue by brdbkr** [organicmaps/organicmaps#9252]
For a few months now OM has shown me unusual small numbers at many, many intersections. I'm not sure what there are meant to be or where the data is coming from because I can't find any matching elements when looking on OSM.org.

![IMG_1180](https://github.com/user-attachments/assets/ff935107-2bda-4f09-8372-8af211afe235)
![IMG_1181](https://github.com/user-attachments/assets/0c39b17d-e12b-4840-b085-5bbfb052d497)

Here's [the area on OSM](https://www.openstreetmap.org/#map=19/34.160641/-84.173777):

![r22bsIA](https://github.com/user-attachments/assets/f965de0d-bb72-42c2-b300-5587cce63418)

Tapping the numbers in OM displays them as addresses. Where are these coming from?

Not only do these numbers not match anything real, but they are also at times incorrect, sometimes showing values that aren't even in the range of nearby addresses.

**Comment by matheusgomesms** [comment]
Is this USA? I believe OM is fetching addresses from external sources for USA since last 2 releases.

**Comment by brdbkr** [comment]
Yes, it's the US.

**Comment by matheusgomesms** [comment]
So yep, very likely this data is coming from TIGER data and not OSM. This was rolled out in the July update.

**Comment by brdbkr** [comment]
See https://github.com/organicmaps/organicmaps/pull/8502

**Comment by vng** [comment]
Yes, not perfect. This is an interim solution according to our roadmap, mainly to enhance US search. The next step is to replace Tiger with the OpenAddresses data.
